go.mod

```diff
@@ -5,42 +5,50 @@ go 1.15
 require (
 	github.com/IBM-Cloud/bluemix-go v0.0.0-20201019071904-51caa09553fb
 	github.com/IBM-Cloud/power-go-client v1.0.53
-	github.com/container-storage-interface/spec v1.2.0
+	github.com/container-storage-interface/spec v1.5.0
 	github.com/davecgh/go-spew v1.1.1
-	github.com/dgrijalva/jwt-go v3.2.0+incompatible
+	github.com/go-openapi/jsonpointer v0.19.5 // indirect
+	github.com/go-openapi/jsonreference v0.19.5 // indirect
+	github.com/go-openapi/swag v0.19.14 // indirect
```

Comment:
see if there is a way to get rid off indirect modules from the go.mod file,

`go mod tidy` may help running with golang 1.17 version.

Reply:
Updated go version to 1.17.
Indirect updates are separated in the go mod file.